### PR TITLE
mark verified phone unstable until fixed

### PR DESCRIPTION
**Pull request by jessingrass**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
I'm marking this test unstable. It appears that the test needs a scroll-to element, based on the screenshots of the failure conditions in meta-ci.

### CODE REVIEW

File: robot/EDA/tests/browser/contacts/verify_preferred_phone.robot [hunk 1]
```diff
@@ -19,6 +19,7 @@ Suite Teardown  Run keywords
 ...   AND       Capture screenshot and delete records and close browser 
 
 *** Test Cases ***
+[tags]  unstable
```
Comment (boakley) [inline robot/EDA/tests/browser/contacts/verify_preferred_phone.robot]: The tags need to be moved after the test name 

```
Verify basic preferred phone functionality
    [tags]  unstable
    Current page should be    Home
    yada yada yada
```

File: robot/EDA/tests/browser/contacts/verify_preferred_phone.robot [hunk 2]
```diff
@@ -1,6 +1,7 @@
 *** Settings ***
 Documentation
 ...     Verify Preferred Phone functionality
+Force Tags      unstable
```
Comment (robertbwatson) [inline robot/EDA/tests/browser/contacts/verify_preferred_phone.robot]: @dashattuck Does #1006 supercede this? It looks like this PR might no longer be required if we're marking the tests unstable there.

Comment (dashattuck) [reply robot/EDA/tests/browser/contacts/verify_preferred_phone.robot]: @robertbwatson - correct this PR is no longer required as #1006 supersedes it.

**Comment by robertbwatson** [comment]
@jessingrass @dashattuck Per Doug's comments above that #1006 supersedes this PR and also marks the tests as unstable, I'm going to go ahead and close this PR. Note also that we are now in a merge freeze until Wednesday afternoon.

**Pull request closed without merging**